.github/workflows/R-CMD-check.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - 61-cran
```

Comment:
This should be removed still; we don't want to clutter up the actions file.

Reply:
I'll keep using the branch at least until submission. I'll remove it afterwards.

Reply:
Second thoughts, let's use a draft pull request to trigger this more cleanly.

```suggestion
```